src/main/webapp/script.js

```diff
@@ -86,13 +87,37 @@ function clearElm(elementID) {
 
 function userResult() {
     clearElm(QUIZ_ELEMENT_ID);
-    const quiz = document.getElementById(QUIZ_ELEMENT_ID);
     quiz.appendChild(createParagraphElement("Based On Your Results, You Should visit..."));
     displayResults(this);
 }
 
 function displayResults() {
+    jsonPlaces = '{ "Places" : [' +
+   '{ "Place":"Paris" , "Currency":"Euro","Language":"Parisian French","Price":"$$$","Food":["Caramels", "Baguette", "Pain Au chocolat", "Pastries", "Chocolate", "Macarons", "Cheese from Laurent Dubois", "Crème Brûlée ", "Éclair", "Croissants" ]},' +
+        '{ "Place":"New York" , "Currency":"US Dollar","Language":"English","Price":"$$$","Food":["Pizza", "Bagels", "Burgers", "Sandwiches", "Ramen", "Food Trucks", "Cheesecake" ]},' +
+        '{ "Place":"Hawaii" , "Currency":"US Dollar","Language":"English, Creole, and Hawaiian Pidgin","Price":"$$","Food":["All-Natural Shave Ice", "Saimin", "Poke", "Luau Stew", "Manapua", "Fish Tacos", "Huli Huli Chicken", "Loco Moco", "Malasadas"]},' +
+        '{ "Place":"Cape Town" , "Currency":"South African Rand","Language":"Afrikaans","Price":"$","Food":["Fish and Chips", "Game Meat", "Gatsby", "Bunny Chow","Bobotie", "Biltong and Droëwors", "Malva Pudding" , "Koeksister"] } ]}';
+    resultObject = JSON.parse(jsonPlaces);
     var h = document.createElement("H3");
-    h.appendChild(document.createTextNode("Paris!"));
+    let place = Math.floor(Math.random() * 4);
+    if(place==0){
+        i=0;
+    }else if(place==1){
+        i=1;
+    }else if(place==2){
+        i=2;
+    }else if(place==3){
+        i=3;
+    }
```

Comment:
looks like i and place are the same random number, so you could remove this chunk and just use place as your index (maybe just rename place as placeIndex?)